### PR TITLE
change 2020 statements from future tense

### PR DESCRIPTION
**Pull request by mscuthbert**
A discussion point: is the entire first section the statement or is the single sentence pledge at the bottom the statement?  If the latter, then it makes sense to update the language above from future to past tense.

### CODE REVIEW

File: _sections/20-statement.md [hunk 1]
```diff
@@ -12,33 +12,29 @@ two versions of the language for several years. While we have developed tools
 and techniques to maintain compatibility efficiently, it is a small but
 constant friction in the development of a lot of code.
 
-We are keen to use Python 3 to its full potential, and we currently accept the
-cost of writing cross-compatible code to allow a smooth transition, but we
-don’t intend to maintain this compatibility indefinitely. Although the
-transition has not been as quick as we hoped, we do see it taking place, with
+We are keen to use Python 3 to its full potential, and though 
+we had been accepting the
```
Comment (takluyver) [inline _sections/20-statement.md]: we accepted

File: _sections/20-statement.md [hunk 2]
```diff
@@ -12,33 +12,29 @@ two versions of the language for several years. While we have developed tools
...
+cost of writing cross-compatible code to allow a smooth transition, we
+did not intend to maintain this compatibility indefinitely. Although the
+transition was not as quick as we hoped, it has taken place, with
```
Comment (takluyver) [inline _sections/20-statement.md]: hmmm... 'has taken place' still feels a bit premature, to be honest.

File: _sections/20-statement.md [hunk 3]
```diff
@@ -12,33 +12,29 @@ two versions of the language for several years. While we have developed tools
...
 if you enjoy the free, first party support for many projects including the Scientific
-Python stack, please start planning to move to Python 3.
+Python stack, please move to Python 3.
 
 For all of these reasons, the following projects have pledged to **drop support
 for Python 2.7 no later than 2020**, coinciding with the Python development
-team's [timeline for dropping support for Python
-2.7](https://www.python.org/dev/peps/pep-0373/#update).
+team's [drop of support for Python
+2.7 on January 1, 2020](https://www.python.org/dev/peps/pep-0373/#update).
```
Comment (takluyver) [inline _sections/20-statement.md]: 'drop of support' is a bit awkward. How about 'following the end of support for Python 2.7 itself at the start of the year'.

File: _sections/20-statement.md (hunk 2 above)
Comment (mscuthbert) [reply _sections/20-statement.md]: How about "it is taking place" as something stronger than "we do see it taking place" but less forceful than the past tense version.

File: _sections/20-statement.md (hunk 3 above)
Comment (mscuthbert) [reply _sections/20-statement.md]: perfect

**Pull request merged**